src/openrct2/Context.cpp

```diff
@@ -305,26 +326,10 @@ namespace OpenRCT2
                 _uiContext->CreateWindow();
             }
 
-            // TODO add configuration option to allow multiple instances
-            // if (!gOpenRCT2Headless && !platform_lock_single_instance()) {
-            //  log_fatal("OpenRCT2 is already running.");
-            //  return false;
-            // }
 
-            _objectRepository = CreateObjectRepository(_env);
```

Comment:
leave these here

Reply:
Core Dump if they aren't above the code getting language.

Reply:
TL;DR This has to exist before Language, because Language depends on it existing.